Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -360,13 +360,13 @@ CGDataProviderRef CGImageGetDataProvider(CGImageRef img) {
 
     const unsigned int stride = CGImageGetBytesPerRow(img);
     const unsigned int size = CGImageGetHeight(img) * stride;
-    woc::unique_iw<unsigned char> buffer(static_cast<unsigned char*>(IwMalloc(size)));
+    std::unique_ptr<unsigned char> buffer(static_cast<unsigned char*>(malloc(size)));
```

Comment:
>std::unique_ptr [](start = 4, length = 15)

This isn't exactly equivalent. unique_iw used IwFree to deallocate memory, while unique_ptr uses operator delete.

We should make sure the alloc and dealloc methods match.

(Also, the right type would be unique_ptr<unsigned char[]>.)